iocage.8

```diff
@@ -498,15 +498,16 @@ UUID/NAME wrapper for
 After invoking
 .Cm exec ,
 specify the jail, any commands to run inside that jail, and any
-arguments for those commands. 
+arguments for those commands.
 .Xr jexec
-also runs commands similar to 
+also runs commands similar to
 .Nm .
 When using
 .Xr jexec
 use the JID instead of the jail name.
-For more info see the manual page for 
+For more info see the manual page for
 .Xr jexec .
+Use -- in front of specified options to prevent iocage from parsing them.
```

Comment:
s/options/command/

Reply:
Why did you mark this resolved? This isn't just options that need it, it's the whole command they specify.
`iocage exec JAIL -- COMMAND` not just options.